less/forms.less

```diff
@@ -764,3 +764,19 @@ select {
     }
   }
 }
+
+// From commonStyles.js
```

Comment:
I am not quite if it should be recovered: https://github.com/brave/browser-laptop/commit/4fb5d5ae8202b5afffff4e3ce106a64d81fbfdb4 CC @bsclifton for a comment